code/modules/mob/mob.dm

```diff
@@ -1061,12 +1061,11 @@ GLOBAL_LIST_INIT(slot_equipment_priority, list( \
 		to_chat(src, "<span class='warning'>You can't respawn as an NPC before the game starts!</span>")
 		return
 
-	if((usr in GLOB.respawnable_list) && (stat==2 || istype(usr,/mob/dead/observer)))
+	if((usr in GLOB.respawnable_list) && (stat == DEAD || istype(usr,/mob/dead/observer)))
 		var/list/creatures = list("Mouse")
-		for(var/mob/living/L in GLOB.alive_mob_list)
-			if(safe_respawn(L.type) && L.stat!=2)
-				if(!L.key)
-					creatures += L
+		for(var/mob/living/simple_animal/L in GLOB.mob_living_list)
```

Comment:
Wouldn't the `alive_mob_list` be less populated than the `mob_living_list` and thus be a better fit? Since observers won't be included in here as well and `alive_mob_list` won't have dead mobs in them anyway